runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -1355,95 +1356,49 @@ scan_hex_helper(J9JavaVM *javaVM, char **cursor, UDATA *value, const char *argNa
 }
 
 /**
- * Wrapper for scan_udata_helper, that provides parsing for memory sizes.
- * User should be able to specify the size in GiBs, MiBs, or KiBs (with G,g,M,m,K,k suffixes) or
- * in bytes (no suffix)
+ * Wrapper for scan_udata_memory_size, that provides readable error messages.
+ * @param cursor address of the pointer to the string to parse for the udata.
+ * @param value address of the storage for the udata to be read.
+ * @param argName string containing the argument name to be used in error reporting.
  * @return true if parsing was successful, false otherwise.
  */
 bool
 scan_udata_memory_size_helper(J9JavaVM *javaVM, char **cursor, UDATA *value, const char *argName)
 {
 	PORT_ACCESS_FROM_JAVAVM(javaVM);
+	UDATA result = scan_udata_memory_size(cursor, value);
```

Comment:
Would you please change `UDATA` to `uintptr_t` for consistency?

Reply:
`UDATA` is defined to be `uintptr_t`. This file opts to use `UDATA` over `uintptr_t`. 
As an example of many, see: https://github.com/eclipse/openj9/blob/1707149cde281821202101158513bd7bbfa20459/runtime/gc_modron_startup/mmparse.cpp#L1261-L1278

Reply:
I know. Historically we used `UDATA` but with introduction of OMR prefer to use `uintptr_t`. We did not replace definitions across OpenJ9 globally (only for code migrated to OMR), however trying to use new definitions in changes and be consistent over function (do not use both even essentially they means the same)

Reply:
Thanks for the explanation. I will make that change.